### PR TITLE
Improve readability of Getting Started guide

### PR DESCRIPTION
**Pull request by soyrice**
New pytest user here! Updated the Getting Started guide to be more readable for new users. Improved grammar, phrasing, etc.

### CODE REVIEW

**Comment by soyrice** [comment]
Done :)

File: doc/en/getting-started.rst [hunk 1]
```diff
@@ -177,22 +160,21 @@ before performing the test function call.  Let's just run it::
     PYTEST_TMPDIR/test_needsfiles0
     1 failed in 0.12 seconds
 
-Before the test runs, a unique-per-test-invocation temporary directory
-was created.  More info at :ref:`tmpdir handling`.
+More info on tmpdir handeling is available at `Temporary directories and files <tmpdir handling>`_.
```
Comment (nicoddemus) [inline doc/en/getting-started.rst]: typo: handeling

**Comment by nicoddemus** [comment]
Thanks @soyrice.

Let's wait for @The-Compiler's review before merging. 👍

**Comment by The-Compiler** [comment]
No time right now thanks to exams, so feel free to go ahead :wink:

**Comment by nicoddemus** [comment]
OK then! 👍

**Pull request merged**